client/dashboard/domains/name-servers/form-input.tsx

```diff
@@ -0,0 +1,50 @@
+import { __experimentalInputControl as InputControl } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { NameServerField, MIN_NAMESERVER_LENGTH } from './types';
+import { validateHostname } from './utils';
+
+export const validateField = ( value: string, index: number ): string => {
```

Comment:
Are we not using any form library in the new dashboard? cc @youknowriad

Reply:
DataForm is the form library. It's a bit young though and if there's something missing let me know.